### DISCUSSION TITLE
Tab type for Group Field

### DISCUSSION THREAD

**Discussion by mrleblanc101** [directus/directus#11536] [Feature Requests]
It would be nice to be able to layout fields inside tabs group. For exemple, SEO fields don't need to be shown with the main form. I could put them in an accordion, but I feel like tabs would be a great addition to directeur and shouldn't be too hard to add.

**Comment by azrikahar** [comment]
Tabs being in a layout has indeed been discussed before, though the immediate discussion I could find which mentioned this briefly would be here: https://github.com/directus/directus/discussions/9161#discussioncomment-1816128 👍

**Reply by benhaynes** [reply]
And here: https://github.com/directus/directus/discussions/3045

**Comment by mrleblanc101** [comment]
Thanks I've followed both issue. I hope Tabs can be added soon-ish as they are easy. Although, Multi-part wizard might be more complex.

**Reply by rijkvanzanten** [reply]
> I hope Tabs can be added soon-ish as they are easy.

Wanna open a PR then? 🙂

**Reply by mrleblanc101** [reply]
Don't take this the wrong way. I was comparing to the Multi-part wizard suggestion. 

I don't know Directus codebase and I have no idea how the core team would want to implement this.

Can you move tabs anywhere ? I'm more talking about "tabbed" pages. Do they want to support both ? Etc...

**Reply by rijkvanzanten** [reply]
hehe gotcha! I think tabs should be implemented as a nested group type, similar to how the current implementation works for the accordion group. This basically means that every field that's in the group will become a "tab", which in turn allows you to use another group to have multiple fields in the same tab. The name of the field will become the name of the tab. This would allow for all sorts of creative uses, like nested tabs, accordions in tabs, any level of nested relational fields, and everything in between 🙂

**Reply by benhaynes** [reply]
I agree with the consistency this gives us, and the logic makes sense. I do think that we might want to wrap this setup with some dedicated UI in the app, so that we remove the confusion of one field per tab (except for the rare case that's intended).

**Comment by mrleblanc101** [comment]
I had suggested this on the YouTube video about grouping that was published not long ago on the channel

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
